cmd/snap/cmd_snap_op.go

```diff
@@ -383,35 +451,55 @@ func showDone(cli *client.Client, chg *client.Change, names []string, op string,
 		}
 		switch op {
```

Comment:
maybe is worth refactoring things with  some separate helpers, and have the op switches in the helpers:
```
if _, ok := notOnlyComps[snap.Name]; ok {
    showDoneSnap(op, snap.Name)
}
if  comps := snapsData.comps[snap.Name]; len(comps) > 0 {
    showDoneComps(op, snap.Name, comps)
}
```

or similar

Reply:
Good point, done now